apps/help-center/help-center.scss

```diff
@@ -70,15 +53,23 @@ body.is-nav-unification {
 
 		svg {
 			position: absolute;
-			height: 24px;
-			width: 24px;
-			top: 0;
+			height: 22px;
+			width: 22px;
+			top: 1px;
 			left: 0;
 			right: 0;
 			bottom: 0;
 			padding: 4px 6px;
 		}
 
+		@media (max-width: 782px) {
+			svg {
+				height: 30px !important;
+				padding: 7px 8px !important;
+				width: 30px !important;
+			}
+		}
```

Comment:
I think this is could be confusing in the future since this styling exists in the WPCOM Admin Bar package - https://github.com/Automattic/jetpack/blob/trunk/projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss#L112-L114

Maybe it would be better to update this in Jetpack rather than overwriting it with `! important` here? Or vice-versa remove the styling from the WPCOM Admin Bar and bake it into the icon here.

Reply:
Agreed. I planned to remove the competing CSS from Jetpack, and use this new code. Since this code currently takes precedence due to the `!important`, we could land Calypso "at will" and remove the dead CSS from Jetpack at leisure, aka when it lands next. I'll raise that Jetpack PR next. How does that plan sound? ☺️

Reply:
Deal! Happy to approve that PR when you got it.

Reply:
Follow-up PR to remove the dead CSS here: https://github.com/Automattic/jetpack/pull/38659